decision_service/dsclientlib/logger/ds_async_batcher.h

```diff
@@ -0,0 +1,123 @@
+#pragma once
+
+#include <string>
+#include <thread>
+#include <sstream>
+
+#include "ds_concurrent_queue.h"
+#include "ds_api_status.h"
+#include "../ds_error_callback_fn.h"
+
+namespace decision_service {
```

Comment:
A naming question: should we call it decision_service?

Reply:
vw_client , vw_client_lib more appropriate?  Other suggestions?  

---
In reply to: [187104723](https://github.com/JohnLangford/vowpal_wabbit/pull/1479#discussion_r187104723) [](ancestors = 187104723)